### PR TITLE
Perpetual layout menu

### PR DESCRIPTION
**Pull request by tiltom**
Resolves https://sovryn.monday.com/boards/2218344956/pulses/2353751954

### CODE REVIEW

**Comment by ShynRou** [comment]
image
I changed a few things from the original design and the previous version:
1. The GSN stays on the right side as, left aligning it on the customize layout button makes no sense
2.  I didn't implement the merged dropdown, as it would need to be new custom component and I don't see the value in that atm. Could be something we should add to the Atomic Components Backlog.
3. I stretched the DataCard components vertically to fill the screen size, except for the Place New Order Card as this one requires a set height.

**Review by soulBit — CHANGES_REQUESTED** [review]
Code and functionality seems ok, but please clean up the build warnings before merging:
image

File: src/app/pages/PerpetualPage/components/LayoutMenu/index.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Switch, Popover } from '@blueprintjs/core';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectPerpetualPage } from '../../selectors';
+import { actions } from '../../slice';
+import { useTranslation } from 'react-i18next';
+import { translations } from 'locales/i18n';
+import arrowDownIcon from 'assets/images/swap/ic_arrow_down.svg';
+
+type LayoutMenuProps = {};
```
Comment (tiltom) [inline src/app/pages/PerpetualPage/components/LayoutMenu/index.tsx:10]: Unnecessary, delete it pls.

File: src/app/pages/PerpetualPage/components/LayoutMenu/index.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          />
+        </div>
+      }
+    >
+      <button className="tw-flex tw-items-center tw-py-1 tw-bg-gray-3 tw-px-5 tw-rounded-lg tw-cursor-pointer tw-select-none tw-transition-opacity hover:tw-bg-opacity-75">
+        <div className="tw-flex tw-flex-row tw-justify-start tw-items-center tw-flex-shrink-0 tw-flex-grow tw-mr-4">
+          {t(translations.perpetualPage.layout.button)}
+        </div>
+        <img
+          className={'tw-w-3 tw-ml-2'}
```
Comment (tiltom) [inline src/app/pages/PerpetualPage/components/LayoutMenu/index.tsx:90]:
```suggestion
          className="tw-w-3 tw-ml-2"
```

**Review by tiltom — COMMENTED** [review]
I cannot request changes on my PRs so I submit these as comments :D

**Comment by ShynRou** [comment]
Fixed issues raised by @tiltom and removed dead variable `connected` @soulBit.
Critical dependency error is in a function that we do not use in the fronend tudor should fix that in the package.
transferAccountLocked should by addressed in #2071.

**Review by tiltom — COMMENTED** [review]
I cannot approve the request so take this as my approval :D

**Review by soulBit — APPROVED** [review]

**Pull request merged**